extensions/wikia/AutoLogin/templates/AutoLogin_Passive.php

```diff
@@ -0,0 +1 @@
+<iframe src="<?= $url; ?>" class="auto-login-module-iframe"></iframe>
```

Comment:
This should be controlled using JS with browser detection.

Reply:
Correct. Will do.

Reply:
@rogatty update in 25bb41ab5e9

Reply:
@harnash soo, this file can be deleted, right?

Reply:
It can: ddead02ddc3